conbench/app/benchmarks.py

```diff
@@ -180,17 +194,38 @@ def show_trends_for_benchmark(bname: TBenchmarkName) -> str:
         # datetimeindex back to float values. Remove 10^15 from these
         # nanoseconds-since-epoch to make the numeric values a little less
         # extreme for printing/debugging.
+
+        remove_outliers_by_iqrdist(df, "svs")
```

Comment:
bug here: no copy before mutation -- this directly affects BMRT cache. fixing in next patch.